docs/modules/ROOT/pages/subscriptions/index.adoc

```diff
@@ -1,18 +1,21 @@
 [[subscriptions]]
 = Subscriptions (Beta)
 
-WARNING: Subscriptions are only available as a beta its API may change in the future. It is not recommended to use subscriptions in production environments.
+WARNING: Subscriptions are only available as a beta, its API may change in the future. It is not recommended to use subscriptions in production environments.
 
-
-GraphQL subscriptions add real-time capabilities to your API. This allows you to listen for changes to the database.
+GraphQL subscriptions add real-time capabilities to your API. Subscriptions allow to listen for changes on the database.
 
 The following subscriptions are available when using `@neo4j/graphql`:
 
 * **xref::subscriptions/events/create.adoc[Create]** - Listen for newly created nodes.
 * **xref::subscriptions/events/update.adoc[Update]** - Listen for changes to existing nodes.
 * **xref::subscriptions/events/delete.adoc[Delete]** - Listen for deleted nodes.
 
-All events will be triggered individually, with one event per node created, updated or deleted, even in events caused by the same mutation. 
+All events will be triggered individually, with one event per node created, updated or deleted, even in events caused by the same mutation.
+Events ordering is guaranteed by default, although this may not be true when using some message brokers.
```

Comment:
Are we sure we want to make this claim? 👀

Reply:
```suggestion
```